### PR TITLE
More gen 3 - profile generated 100%

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] login/saga -> login.js
- [x] login/provion-helpers -> constants/login
- [x] generate profile

### CODE REVIEW

File: shared/profile/pgp/finished-generating-pgp.desktop.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ class FinishedGeneratedPgp extends Component<Props, State> {
         <PlatformIcon style={styleIcon} platform="pgp" overlay="icon-proof-success" />
         <Text style={styleTitle} type="Header">Here is your unique public key!</Text>
         <Box style={{...globalStyles.flexBoxRow, alignSelf: 'stretch'}}>
-          <textInput style={stylePgpKeyString} readOnly={true}>{this.props.pgpKeyString}</textInput>
+          <textinput style={stylePgpKeyString} readOnly={true}>{this.props.pgpKeyString}</textinput>
```
Comment (chrisnojima) [inline shared/profile/pgp/finished-generating-pgp.desktop.js]: react complains that this should be lowercase

**Comment by chrisnojima** [comment]
@keybase/react-hackers

File: shared/actions/profile-gen.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
+// @flow
+// NOTE: This file is GENERATED from json files in actions/json. Run 'yarn build-actions' to regenerate
+/* eslint-disable no-unused-vars,prettier/prettier */
+
+import {type ReturnType} from '../constants/types/more'
+import * as Constants from '../constants/profile'
+import * as RPCTypes from '../constants/types/flow-types'
+import * as More from '../constants/types/more'
```
Comment (buoyad) [inline shared/actions/profile-gen.js]: are generated action files linted?
```
$ yarn run lint-specific .

/Users/dan/go/src/github.com/keybase/client/shared/actions/profile-gen.js
  5:31  error  '../constants/types/more' imported multiple times  import/no-duplicates
  8:23  error  '../constants/types/more' imported multiple times  import/no-duplicates

✖ 2 problems (2 errors, 0 warnings)
```

Comment (chrisnojima) [reply shared/actions/profile-gen.js]: yeah i fixed this in some future prs

**Pull request merged**